src/Text/Text.tsx

```diff
@@ -64,11 +99,31 @@ export const Text: FC<TextProps> = forwardRef<HTMLElement, TextProps>(
 
 Text.displayName = 'Text'
 
+const isNewTypo = (value: string): value is Typo => {
+  return Object.keys(fontStyleMapping).includes(value)
+}
+
+const generateTypo = (typo: Typo): FlattenSimpleInterpolation => {
+  let styles
+
+  Object.keys(fontStyleMapping).forEach((font) => {
+    if (typo === font) {
+      styles = fontStyleMapping[font]
+    }
+  })
+
+  return css`
+    ${styles}
+  `
+}
+
 const Container = styled(Box)<IText>(
   ({ align, color, cursor, typo }) => css`
-    /** PREDEFINED TYPOGRAPHY STYLES */
+    /** TYPOGRAPHY STYLES */
+
+    ${isNewTypo(typo) && generateTypo(typo)}
```

Comment:
You should be able to inline `fontStyleMapping` here
```suggestion
    ${isNewTypo(typo) && fontStyleMapping[font]}
```